.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
 	"env": {
-		"node": true
+		"node": true,
+    "es6": true
 	},
 	"plugins": [
 		"nodeca"
```

Comment:
Thanks for the feedback, made the changes to the package.json, but when trying to remove the nodeca plugin, I got a lot of linting errors for the entire module. Did you want me to copy the eslintrc file from webpack in its entirety?